vaadin-icons-flow-parent/pom.xml

```diff
@@ -12,7 +12,6 @@
     <modules>
         <module>vaadin-icons-flow</module>
         <module>vaadin-icons-testbench</module>
-        <module>vaadin-icons-flow-demo</module>
```

Comment:
Moved the IconView class from the demo project to vaadin-icons-flow-integration-tests because that's the only place it's used. The demo project was removed as obsolete.

Reply:
They should all be removed as the component demos are nowadays elsewhere. Many of the old demo modules are still used by the integration tests though so they need to be manually moved to the integration test module (like I've done in this case).